lms/djangoapps/instructor/utils.py

```diff
@@ -185,16 +183,32 @@ def collect_ora2_data(course_id):
 
 
 # pylint: disable=invalid-name
-def ora2_data_queries():
+def ora2_data_queries(is_anonymous):
     """
     Wraps a raw SQL query which retrieves all ORA2 responses for a course.
     """
-
-    RAW_QUERY = """
+    if is_anonymous:
+        RAW_QUERY_3 = """
+        `student`.`student_id` AS `anonymized_student_id`,
```

Comment:
Nit: This doesn't have to be triple-quoted or split across lines, since it's a single-line string.